pkg/network/ebpf/c/tracer-shared-maps.h

```diff
@@ -0,0 +1,10 @@
+#ifndef __TRACER_SHARED_MAPS_H
+#define __TRACER_SHARED_MAPS_H
+
+#include "map-defs.h"
+
+// Maps a connection tuple to latest tcp segment we've processed. Helps to detect same packets that travels multiple
+// interfaces or retransmissions.
+BPF_HASH_MAP(connection_states, conn_tuple_t, u32, 0)
```

Comment:
the separation of the maps was intended to make usm as the owner of them and to reduce the requirment of networks team approval for classification changes